app/script_variant.py

```diff
@@ -65,6 +65,40 @@ def romanize_japanese_name_by_name_dict(word):
     return [word]
 
 
+def romanize_japanese_name_by_name_dict(word, is_create=True):
+    """
+    This method romanizes japanese name by using name dictionary.
+    If word isn't found in dictionary, this method doesn't
+    apply romanize.
+    This method can return multiple romanizations.
+    (because there are multiple ways to read the same kanji name in japanese)
+    Returns:
+        [romanized_jp_name, ...]
+    """
+    if not word:
+        return ['']
+
+    words = []
+    for index in xrange(1, len(word)):
+        # split word because query word may not contains white space
```

Comment:
This is still not clear. Even if the query doesn't contain white space, it would work fine if the query is "山田". Please explain what exactly is the case you want to handle, maybe with an example.